modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -1,31 +1,29 @@
 = Domain Support for Mule 4 API Proxies
 
-Mule runtime allows sharing common resources among other Mule applications by defining these resources within a Mule Domain, and declaring other Mule applications to inherit this domain.
-This way, all resources declared in your domain, can be accessed by all Mule applications using it. +
-More information about this can be found in the runtime's xref:4.1@mule-runtime::shared-resources.adoc[Shared Resources] topic.
+Mule runtime engine enables you to share common resources among other Mule applications by defining these resources within a Mule Domain. Other Mule applications can then inherit this domain.
 
-Since each auto-generated proxy is a pre-configured Mule application, you can also define multiple API Proxies under the same domain. +
+This way, all of your Mule applications can access the xref:4.1@mule-runtime::shared-resources.adoc[resources]declared in your domain. +
```

Comment:
It's not for all Mule applications, but the ones configured to leverage it.

Reply:
Done! Thanks!